README.md

```diff
@@ -1 +1,184 @@
+
+```
+docker run -e VISION-FACE=True -v localstorage:/datastore -p 80:5000 deepquestai/deepstack
+```
+
+### Face Registration
+The face Registration API goal's is to allow you to enroll a person's name and image on Deepstack server for future use like face recognition. The face recognition API is dependent of this API.
+
+To use this API, you need to enable the face API when starting DeepStack with docker through this command
+
+```
+docker run -e VISION-FACE=True -v localstorage:/datastore -p 80:5000 deepquestai/deepstack
+```
+
+### Face Deregistration
+The face deregistration API goal's is to allow you to erase a person's name and image from Deepstack server. 
+
+To use this API, you need to enable the face API when starting DeepStack with docker through this command
+
+```
+docker run -e VISION-FACE=True -v localstorage:/datastore -p 80:5000 deepquestai/deepstack
+```
+
+### Face Recognition
+The face recognition API goal's is to identify a face that has been previously register on Deepstack server, when it fails to recognize a face it portaits it as an **Unknown** face in the live stream.
+To use this API, you need to enable the face API when starting DeepStack with docker through this command
+
+
+```
+docker run -e VISION-FACE=True -v localstorage:/datastore -p 80:5000 deepquestai/deepstack
+```
+
+### Custom detection
+The custom detection API allows you to perform detection on personal/ specific object. To acheive this, your must prepare your own dataset, trained model , deplayed model and store these elements into a folder.
+
+To activate this API, you need to enable the detection API when starting DeepStack with docker through this command
+
+```
+docker run -v /path-to/custom-models-folder:/modelstore/detection -p 80:5000 deepquestai/deepstack
+```
+
+To test the custom detection API, you can make use of  a custom model built by DeepStack called **[ActionNet](https://github.com/OlafenwaMoses/DeepStack_ActionNET)** to detection human actions such as eating, reading, 
+calling etc.
+
+
+# Run DeepStack-CameraUI with Webhook
+A webhook is a lightweight API that power one-way data transmission and trigger by event. In our case, this API is trigger when the webhook URL is defined,
+this webhook goal's is to transfer in real-time all the detections captured through any above mentioned APIs to a recipient.
+
+This API is is dependent to few components which are:
+- **Webhook URL**: The receiver address, it can be a URL or an email address
+- **Webhook API Key**: the webhook api personal key , it is optional
+- **Webhook Forward Images**:  a function to send detection images to the receipient. it is optioanal.
+
+
+To run DeepStack-CameraUI with webhook, kindly install [Flask](https://flask.palletsprojects.com/en/2.0.x/installation/) and run the following steps:
+
+1. Move to the src folder and run the streamlit script with `streamlit run app.py`
```

Comment:
In this section,
- you don't need mention they should run the streamlit application again
- Instead, tell them to run install the flask app dependencies first in the scripts folder
- Run the flask app
- Specify the flask app URL as webhook and then run the DeepStack-CameraUI app